app/src/main/res/layout/onboarding_language_dropdown_item.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is there any value in making this a constraint layout vs. a frame layout (for performance)?

Reply:
No, and I have replaced it with FrameLayout.